config/vector.yaml

```diff
@@ -132,3 +145,15 @@ sinks:
     inputs:
       - vector_metrics
       - node_metrics
+
+  sdk_sink:
+    type: kafka
+    encoding:
+      codec: json
+      except_fields:
+        - "path"
+        - "source_type"
+    inputs:
+      - "sdk_transformed"
+    bootstrap_servers: kafka0:29092
+    topic: hyper-sdk-logs
```

Comment:
```suggestion
    topic: hyper-sdk-logs
    key_field: "{{ .merchant_id }}{{ .payment_id }}"
```

Reply:
https://vector.dev/docs/reference/configuration/sinks/kafka/#key_field

Reply:
This is giving an error. Had to revert for now